stdlib/2/os/path.pyi

```diff
@@ -99,20 +107,13 @@ elif sys.version_info >= (3, 5):
 # NOTE: Empty lists results in '' (str) regardless of contained type.
```

Comment:
Remove this comment since it was talking about the stub for `commonprefix`. Might be useful to add this comment to `genericpath.pyi` though.